data/resources/ui/login.ui

```diff
@@ -240,6 +240,65 @@
                                 </style>
                               </object>
                             </child>
+                            <child>
+                              <object class="GtkBox">
+                                <property name="orientation">vertical</property>
+                                <property name="spacing">9</property>
+                                <child>
+                                  <object class="GtkStack" id="code_resend_stack">
+                                    <child>
+                                      <object class="GtkStackPage">
+                                        <property name="name">enabled</property>
+                                        <property name="child">
+                                          <object class="GtkButton">
+                                            <property name="action-name">login.resend-auth-code</property>
+                                            <property name="margin-end">12</property>
+                                            <property name="margin-start">12</property>
+                                            <child>
+                                              <object class="GtkLabel" id="code_resend_label">
```

Comment:
Why using a separate label here instead of just modifying the GtkButton::label property?

Reply:
Good catch. This was indeed a leftover from my experiments putting the timeout in the 2nd line of the button's text. But that didn't look very appealing. 
Fixed.